### PR TITLE
Alpha phase 3 part mcmclxviii fixes

### PR DESCRIPTION
**Pull request by baughj**
## Description
Fixes:

HS-728: equipping armor doesn't show stat changes
HS-761: confusion occurring when trying to equip items over equip weight limit
HS-764: allow other items or gold to be picked up under items or gold that cannot be picked up
HS-1091: monsters should be impacted by sleep/freeze/paralyze
HS-972: exchange must be toggled on/off for new characters
HS-828: can't learn a spell with exactly the right amount of gold

### CODE REVIEW

File: hybrasyl/ControlMessages/ControlMessage_Spawn.cs [hunk 1]
```diff
@@ -20,8 +20,10 @@ private void ControlMessage_MonolithControl(HybrasylControlMessage message)
             var monster = (Monster)message.Arguments[0];
             var map = (Map)message.Arguments[1];
 
-            // Don't handle control messages for dead/removed mobs
-            if (!monster.Condition.Alive || monster.DeathProcessed || monster.Id == 0 || monster.Map == null) return;
+            // Don't handle control messages for dead/removed mobs, or mobs that cannot move or attack
+            if (!monster.Condition.Alive || monster.DeathProcessed ||
```
Comment (norrismiv) [inline hybrasyl/ControlMessages/ControlMessage_Spawn.cs:24]: how is condition blind handled? they cannot walk, but can still attack

Comment (Caeldeth) [reply hybrasyl/ControlMessages/ControlMessage_Spawn.cs:24]: blind is no walk, no spells

Comment (norrismiv) [reply hybrasyl/ControlMessages/ControlMessage_Spawn.cs:24]: yeah, but how is it being handled for control messages

Comment (baughj) [reply hybrasyl/ControlMessages/ControlMessage_Spawn.cs:24]: fixed, blind will work similar to usda now

**Review by norrismiv — APPROVED** [review]

**Pull request merged**